qiskit/ignis/verification/accreditation/qotp.py

```diff
@@ -0,0 +1,250 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable= no-member
+"""
+Applies a Quantum One Time Pad (QOTP)
+
+This is essentially the same protocol as used in
+randomized compiling, but follows the methods in
+
+https://iopscience.iop.org/article/10.1088/1367-2630/ab4fd6
```

Comment:
Can this reference be moved to one of the function doc strings so that it is shown in docs?